libs/core/src/components/pds-select/test/pds-select.e2e.ts

```diff
@@ -0,0 +1,42 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('pds-select', () => {
+  it('renders toggle of disabled state', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pds-select></pds-select>');
+    const component = await page.find('pds-select');
+    expect(component).toHaveClass('hydrated');
+
+    const element = await page.find('pds-select >>> select');
+    let value = await element.getProperty('disabled');
+    expect(value).toBe(false);
+
+    component.setProperty('disabled', true);
+    await page.waitForChanges();
+    value = await element.getProperty('disabled');
+    expect(value).toBe(true);
+  });
+
+  it('toggles an error state', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<pds-select error-message="This is error message"></pds-select>');
+    const component = await page.find('pds-select');
+    expect(component).toHaveClass('hydrated');
+
+    const element = await page.find('pds-select >>> .pds-select__error-message');
+    expect(element.textContent).toEqual('This is error message');
+  });
+
+  it('renders with options', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<pds-select><option value="0">Option 1</option><option value="1">Option 2</option></pds-select>');
+    const select = await page.find('pds-select >>> select');
+
+    const options = await select.findAll('option');
+    expect(options.length).toBe(2);
+    expect(options[0].textContent).toBe('Option 1');
+    expect(options[1].textContent).toBe('Option 2');
+  });
```

Comment:
You are missing a test for `value` that would show the behavior of selecting an option on render. 

These E2E tests should be written as if an end-user is using them in the browser. We should `click` functions and check the expectation of the interaction. I do not see any of that here.